bedrock/firefox/templates/firefox/personal.html

```diff
@@ -37,6 +37,9 @@
     <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
       {{ high_res_img('firefox/android/btn-google-play.png', {'alt': 'Get it on Google Play', 'width': '172', 'l10n': True}) }}
     </a>
+    <small class="download-other">
+      <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+    </small>
```

Comment:
Yes the contrast for this one is pretty dark. But not sure white would be okay because that is the colour of text? How about the link colour used in https://www.mozilla.org/en-US/firefox/os/2.0/. It's slightly off white.

<img width="209" alt="screenshot 2015-09-14 15 46 42" src="https://cloud.githubusercontent.com/assets/190703/9860100/0db8b310-5af8-11e5-9902-c5cd17c28908.png">